clustering/src/main/java/io/hyperfoil/clustering/ControllerVerticle.java

```diff
@@ -625,14 +629,52 @@ private void persistRun(Run run) {
             jGenerator.writeFieldName("info");
             jGenerator.writeRawValue(info.encode()); // writeObjectField() was encoding info as a POJO not json
 
-            JsonWriter.writeJson(run.statisticsStore, jGenerator, false);
+            jGenerator.writeFieldName("failures");
```

Comment:
Could we have this code in `JsonWriter`? Actually the whole code about creating the `all.json` would better fit there, just passing `info` as argument.